test/jdk/java/lang/Math/HyperbolicTests.java

```diff
@@ -972,6 +973,46 @@ static int testTanh() {
         return failures;
     }
 
+    /**
+     * Test accuracy of Math.tanh intrinsic using StrictMath.tanh as the reference.
+     * The specified accuracy is 2.5 ulps.
+     *
+     */
+    static int testTanhIntrinsicWithReference() {
+        double b1 = 0.02;
+        double b2 = 5.1;
+        double b3 = 55 * Math.log(2)/2; // ~19.062
```

Comment:
Probably better to use StrictMath.log here or, better use, precompute the value as a constant and document its conceptual origin.